forge-gui/res/cardsfolder/f/flamestick_courier.txt

```diff
@@ -3,10 +3,7 @@ ManaCost:2 R
 Types:Creature Goblin
 PT:2/1
 K:You may choose not to untap CARDNAME during your untap step.
-A:AB$ Pump | Cost$ 2 R T | ValidTgts$ Creature.Goblin | TgtPrompt$ Select target Goblin creature | RememberTargets$ True | AILogic$ ContinuousBonus | SpellDescription$ Target Goblin creature gets +2/+2 and has haste for as long as CARDNAME remains tapped. | StackDescription$ SpellDescription
-S:Mode$ Continuous | Affected$ Creature.IsRemembered | AddPower$ 2 | AddToughness$ 2 | AddKeyword$ Haste
-T:Mode$ Untaps | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ ClearRemembered | Static$ True
-SVar:ClearRemembered:DB$ Cleanup | ClearRemembered$ True
+A:AB$ Pump | Cost$ 2 R T | ValidTgts$ Creature.Goblin | TgtPrompt$ Select target Goblin creature | NumAtt$ 2 | NumDef$ 2 | KW$ Haste | Duration$ UntilUntaps | AILogic$ ContinuousBonus | SpellDescription$ Target Goblin creature gets +2/+2 and has haste for as long as CARDNAME remains tapped. | StackDescription$ SpellDescription
```

Comment:
Maybe some of these StackDescriptions might be nice with REP for the target?

Obviously just window dressing

Reply:
not a huge fan
it seems to remove targeting information when theres a matching localization